client/src/components/forms/DepositBoxForm/useSum.ts

```diff
@@ -0,0 +1,38 @@
+import { AmountsKeys } from "@/utils";
+
+const moneyValues = {
+  '1gr': 0.01,
+  '2gr': 0.02,
+  '5gr': 0.05,
+  '10gr': 0.1,
+  '20gr': 0.2,
+  '50gr': 0.5,
+  '1zł': 1,
+  '2zł': 2,
+  '5zł': 5,
+  '10zł': 10,
+  '20zł': 20,
+  '50zł': 50,
+  '100zł': 100,
+  '200zł': 200,
+  '500zł': 500,
+  EUR: 4.71,
+  GBP: 5.37,
+  USD: 4.33,
+};
+export type moneyValuesType = typeof moneyValues;
+
+export const useSum = () => {
```

Comment:
It's normal function, not react hook, so name 'useSum' is misleading. I don't know about further implementation, but you could just export moneyValues and sum function separately.